test/test_layout_std.py

```diff
@@ -3,6 +3,7 @@
 # Released under the terms of the 2-clause BSD license.
```

Comment:
Don't you need to update blspec tests as well?

Reply:
Not really, nowhere in the blsspec test do we hardcode "gentoo". Type 1 entries are in a directory named by the ENTRY_TOKEN always, and type 2 (UKI) entries are prefixed by the ENTRY_TOKEN when systemd's kernel-install is used. Only when we use non-systemd installkernel do we use "gentoo" as the prefix since we don't have the ENTRY_TOKEN available then.

I suppose I could add some new tests for this case though.

Reply:
> I suppose I could add some new tests for this case though.

Test case added.